packages/console/src/pages/Applications/components/SdkSelector/index.tsx

```diff
@@ -7,27 +7,23 @@ import tada from '@/assets/images/tada.svg';
 import Button from '@/components/Button';
 import Card from '@/components/Card';
 import RadioGroup, { Radio } from '@/components/RadioGroup';
-import { SupportedJavascriptLibraries } from '@/types/applications';
+import { SupportedSdk } from '@/types/applications';
 
 import * as styles from './index.module.scss';
 
 type Props = {
   className?: string;
-  libraryName?: SupportedJavascriptLibraries;
+  sdks: readonly SupportedSdk[];
+  selectedSdk: SupportedSdk;
   onChange?: (value: string) => void;
   onToggle?: () => void;
 };
 
-const LibrarySelector = ({
-  className,
-  libraryName = SupportedJavascriptLibraries.React,
-  onChange,
-  onToggle,
-}: Props) => {
+const SdkSelector = ({ className, sdks, selectedSdk, onChange, onToggle }: Props) => {
   const [isFolded, setIsFolded] = useState(false);
   const { t } = useTranslation(undefined, { keyPrefix: 'admin_console' });
 
-  const librarySelector = useMemo(
+  const sdkSelector = useMemo(
```

Comment:
what's the point of wrapping this into `useMemo` instead of rendering directly?

Reply:
Well, it started from an earlier comment fromo @simeng-li , he recommended that we should extract long tsx element structure into several `useMemo` blocks, and use the memoized object instead, for a cleaner output.
But basically it's all the same, just different structure of code.